prototype/distributor/src/ServerHandler.java

```diff
@@ -2,30 +2,40 @@
 
 import com.google.protobuf.ByteString;
 import com.google.protobuf.InvalidProtocolBufferException;
-import com.google.protobuf.Message;
 import io.netty.buffer.ByteBuf;
 import io.netty.channel.Channel;
 import io.netty.channel.ChannelHandlerContext;
 import io.netty.channel.ChannelInboundHandlerAdapter;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.Map;
-import prototype.distributor.SessionData;
-import prototype.distributor.UsersData;
-import prototype.networkProtocol.MessageManager;
+import network_protocol.MessageManager;
 import prototype.protobuf.Control;
 import prototype.protobuf.General;
 import prototype.protobuf.Session;
 
+import java.util.Date;
+
+/* Things I did that perhaps I shouldn't have
+    - Updated the proto methods (Seemed way redundant to create the outer packet manually every time - wastes space)
+*/
+
+/*Questions I have
+    - why do we do ByteString.copyFrom instead of just converting to bytestring directly
+    - ChannelActive is called twice when I run my testing client
+        - The things the client sends are only handled by one instance, so it is not *Currently* a problem
+        - Why does this happen?
+        - Will this have any future consequences?
+*/
+
 public class ServerHandler extends ChannelInboundHandlerAdapter {
     private Channel channel;
     private String selfName = "";
-    private MessageManager mgr = new MessageManager();
+    private MessageManager msgmgr = new MessageManager();
+    private DatabaseHandler dbmgr = new DatabaseHandler("Users.db");     //Can change where the database is stored later
```

Comment:
if not already present, add `*.db` to gitignore to be safe

Reply:
Is a case by case thing? Or can you configure git to ignore all.db files? I did not push the database itself.